src/main/native/include/sysid/view/AnalyzerPlot.h

```diff
@@ -43,17 +44,29 @@ class AnalyzerPlot {
    * Sets the raw data to be displayed on the plots.
    */
   void SetData(const Storage& data, const std::vector<double>& ff,
-               AnalysisType type);
+               AnalysisType type, std::atomic<bool>& abort);
 
   /**
    * Displays voltage-domain plots.
+   *
+   * @return if plots aren't in the loading state
    */
-  void DisplayVoltageDomainPlots();
+  bool DisplayVoltageDomainPlots();
 
   /**
    * Displays time-domain plots.
+   *
+   * @return if plots aren't in the loading state
```

Comment:
So the function never returns if the plots are in the loading state? Did you mean to say it returns true if the plots aren't in the loading state?